### ISSUE TITLE
Do you need a new logo?

### ISSUE THREAD

**Issue by nunojesus** [z-pattern-matching/z#45]
Hello, I like to contribute with graphic designs, in opensource projects, so if you need something from me like a new logotype or something else just say it, Im here ready to help you.

Greetings!
Nuno Jesus

**Comment by nunojesus** [comment]
Hi See if you like one of this logos.
![z](https://user-images.githubusercontent.com/34600369/40591915-b290b550-6210-11e8-8ece-b4c03f4dbde4.png)

**Comment by leonardiwagner** [comment]
hi @nunojesus , my only concern about the right side logos is about small printing (as favico in website), so I mostly enjoyed the left side ones, my personal taste was about the grey one.

But is up to you choosing what's better, then feel free to update logos via PR, or send here them I update as well, thank you for your work, well done! :clap:

**Comment by nunojesus** [comment]
Thanks for your words, Im trying to create more designs for your project and I will put them here soon. Greetings!

**Comment by nunojesus** [comment]
Hello,
I create other design for the logo, I put diferent colors, see if you like it, if you want other color just say it.

Greetings!

![z2](https://user-images.githubusercontent.com/34600369/41043494-f41cfbea-699b-11e8-94a1-8b6d53d5613d.png)

**Comment by leonardiwagner** [comment]
Hey @nunojesus , I still prefer the 5th option from first art, what do you think about it?

**Comment by nunojesus** [comment]
Ok @leonardiwagner , I will prepair the PNG files, do you have any request?

**Comment by leonardiwagner** [comment]
@nunojesus no, they are great, feel free to do your best, thanks! :smiley:

**Comment by nunojesus** [comment]
Hello, how are you?
I was working in the design of your logo, I decide to make the logo that you like in a square, I think it looks better, and I made some color variations and gradiant background.
See if you like one of this.

Greetings!

Nuno Jesus
![zquadrado222](https://user-images.githubusercontent.com/34600369/41488477-57b1937e-70e4-11e8-812a-1b0c0495da7a.png)

**Comment by leonardiwagner** [comment]
@nunojesus woa! Thanks for more options, but I still think the first one is good because most of our logo display is small.. so those details won't be displayed 😞 , let's use the first one so?
